src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ CHANGELOG
 ---
  * Add the `normalizer` option to the `Unique` constraint
  * Add `Validation::createIsValidCallable()` that returns true/false instead of throwing exceptions
+ * Add optional `conditionExpression` to the `Count` constraint to count only items matching the given symfony expression language
+
+   ```php
```

Comment:
This example should be in the Symfony docs repo not in the changelog

Reply:
done https://github.com/symfony/symfony-docs/pull/15275